unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
>  This is apparently true for MSVC2013 build system, or how it can build stuff at all?

indeed the header is ran through the preprocessor in ```genhdr.targets```